<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>frmButtonDefence</class>
 <widget class="QWidget" name="frmButtonDefence">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QLabel" name="labMap">
     <property name="frameShape">
      <enum>QFrame::Box</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Sunken</enum>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="frame">
     <property name="maximumSize">
      <size>
       <width>100</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="frameShape">
      <enum>QFrame::Box</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Sunken</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QPushButton" name="btnCircle">
        <property name="text">
         <string>圆形</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnPolice">
        <property name="text">
         <string>警察</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnBubble">
        <property name="text">
         <string>气泡</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnBubble2">
        <property name="text">
         <string>气泡2</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnMsg">
        <property name="text">
         <string>消息</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnMsg2">
        <property name="text">
         <string>消息2</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="Line" name="line">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnArming">
        <property name="text">
         <string>布防</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnDisarming">
        <property name="text">
         <string>撤防</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnAlarm">
        <property name="text">
         <string>报警</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnBypass">
        <property name="text">
         <string>旁路</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnError">
        <property name="text">
         <string>故障</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="ckCanMove">
        <property name="text">
         <string>可移动</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
